Add links to Microsoft Learn resources for System app (#558)
microsoft/BCApps: CONTRIBUTING.md

    Once the issue is created on GitHub, it still needs to be approved. Once approved, the rest of the process is identical to the process described in **A)**.

## Get to Know the System App

Before updating the System Application, familiarize yourself with the following sites on Microsoft Learn:

> - [Module architecture](https://learn.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/devenv-blueprint)
> - [Get started with modules](https://learn.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/devenv-blueprint)
> - [Set up your development environment](https://learn.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/devenv-set-up-an-environment)
> - [Create a new module](https://learn.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/devenv-new-module)
> - [Create a .NET wrapper module](https://learn.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/devenv-create-a-wrapper-module)
> - [Change a module](https://learn.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/devenv-change-a-module)

Reviewer: ```suggestion
- [Module architecture](https://learn.microsoft.com/dynamics365/business-central/dev-itpro/developer/devenv-blueprint)
- [Get started with modules](https://learn.microsoft.com/dynamics365/business-central/dev-itpro/developer/devenv-blueprint)
- [Set up your development environment](https://learn.microsoft.com/dynamics365/business-central/dev-itpro/developer/devenv-set-up-an-environment)
- [Create a new module](https://learn.microsoft.com/dynamics365/business-central/dev-itpro/developer/devenv-new-module)
- [Create a .NET wrapper module](https://learn.microsoft.com/dynamics365/business-central/dev-itpro/developer/devenv-create-a-wrapper-module)
- [Change a module](https://learn.microsoft.com/dynamics365/business-central/dev-itpro/developer/devenv-change-a-module)
```

Author: Wow, didn't know it was possible to remove the language in the url. Excellent suggestion

    Once the issue is created on GitHub, it still needs to be approved. Once approved, the rest of the process is identical to the process described in **A)**.

> ## Get To Know System App

Reviewer 2: I'm not a native speaker, but without the article, it sounds weird to me.
```suggestion
## Get To Know The System App
```

Do we need to capitalize every word?
```suggestion
## Get to Know the System App
```

> There are several chapters on Microsoft Learn that explains several aspects of the System app.  
> Here are some:

Reviewer 2: It's twice "several", and instead of "some", you list all of them. That's why I suggest:
```suggestion
Before updating the System Application, familiarize yourself with the following sites on Microsoft Learn:
```

> - [Module Architecture](https://learn.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/devenv-blueprint)
> - [Get started with modules in the System Application](https://learn.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/devenv-blueprint)
> - [Set Up an Environment for Developing a Module](https://learn.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/devenv-set-up-an-environment)
> - [Create a new module in the System Application](https://learn.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/devenv-new-module)
> - [Create a .NET Wrapper Module](https://learn.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/devenv-create-a-wrapper-module)
> - [Change a module in the System Application](https://learn.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/devenv-change-a-module)

Reviewer 2: I'd stick to the TOC view:
![image](https://github.com/microsoft/BCApps/assets/34504100/89999245-0d88-415a-b7a7-d5c61cf8280c)

```suggestion
- [Module architecture](https://learn.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/devenv-blueprint)
- [Get started with modules](https://learn.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/devenv-blueprint)
- [Set up your development environment](https://learn.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/devenv-set-up-an-environment)
- [Create a new module](https://learn.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/devenv-new-module)
- [Create a .NET wrapper module](https://learn.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/devenv-create-a-wrapper-module)
- [Change a module](https://learn.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/devenv-change-a-module)
```

Author: @NKarolak  You mean like this? 😉
![image](https://github.com/microsoft/BCApps/assets/34970096/994ea979-7615-4e15-ac09-09861440ce97)

Reviewer 2: It's only about the casing: Please look again at my code suggestion above, the green part that did not make it into your screenshot, except a thin green bottom line ;-)

Author: Crap, you are right again! 😊
